File: exercicios em C/q56.cpp
#include <stdio.h>

int main(void){
	int i, n, peso, altura, idade, mediaP, mediaA, mediaI, maA, maP, maI, meA, meP, meI;
	printf("\n\nEste programa determina a media, maior e menor, peso, idade e altura de n pessoas.");
	printf("\n\nDigite a quantidade de pessoas a qual voce ira compara peso, idade, e altura: ");
	scanf("%d", &n);
	for(i = 1; i <= n; i++){
		printf("\n\nDigite o peso da %d pessoa: ", i);
		scanf("%d", &peso);
		printf("\nDigite a altura da %d pessoa: ", i);
		scanf("%d", &altura);
		printf("\nDigite a idade da %d pessoa: ", i);
		scanf("%d", &idade);
		mediaP = mediaP + peso;
		mediaA = mediaA + altura;
		mediaI = mediaI + idade;
		if(i == 1){
			maA = altura;
			maP = peso;
			maI = idade;
			meA = altura;
			meP = peso;
			meI = idade;
		}
		if(maA < altura){
			maA = altura;
		}
		else if(meA > altura){
			meA = altura;
		}
		if(maP < peso){
			maP = peso;
		}
		else if(meP > peso){
			meP = peso;
		}
		if(maI < idade){
			maI = idade;
		}
		else if(meI > idade){
			meI = idade;
		}
	}
	mediaP = mediaP / n;
	mediaA = mediaA / n;
	mediaI = mediaI / n;
	printf("\n\nAs medias sao: media de peso %d, media de altura %d, e media de idade %d.", mediaP, mediaA, mediaI);
	printf("\nA maior altura e %d, e a menor altura e %d, o maior peso e %d, e o menor peso e %d, a maior idade e %d, e a menor idade e %d.", maA, meA, maP, meP, maI, meI);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}

File: exercicios em Python/q56.py
import os

print("\nEste programa determina a media, maior e menor, peso, idade e altura de n pessoas.")
n = int(input("\nDigite a quantidade de pessoas a qual voce ira compara peso, idade, e altura: "))
i = 1
mediaP = 0
mediaI = 0
mediaA = 0
while i <= n:
        print("\n\nDigite o peso da ",i," pessoa: ")
        peso = float(input(""))
        print("\nDigite a altura da ",i," pessoa: ")
        altura = float(input(""))
        print("\nDigite a idade da ",i," pessoa: ")
        idade = float(input(""))
        mediaP = int(mediaP + peso)
        mediaA = int(mediaA + altura)
        mediaI = int(mediaI + idade)
        if i == 1:
                maA = altura
                maP = peso
                maI = idade
                meA = altura
                meP = peso
                meI = idade

        if maA < altura:
                maA = altura

        elif meA > altura:
                meA = altura

        if maP < peso:
                maP = peso

        elif meP > peso:
                meP = peso

        if maI < idade:
                maI = idade

        elif meI > idade:
                mel = idade

        i = i + 1

meI = idade
mediaP = int(mediaP / n)
mediaA = int(mediaA / n)
mediaI = int(mediaI / n)
print("\n\nAs medias sao: media de peso ",mediaP,", media de altura ",mediaA,", e media de idade ", mediaI)
print("\nA maior altura e ",maA,", e a menor altura e ",meA,", o maior peso e ",maP,", e o menor peso e ",meP,", a maior idade e ",maI,", e a menor idade e ", meI)
print("\nObrigado por utilizar nosso sistema!")
os.system("pause")
